compiler/msgs.nim

```diff
@@ -64,6 +64,8 @@ type
     errVarForOutParamNeeded,
     errPureTypeMismatch, errTypeMismatch, errButExpected, errButExpectedX,
     errAmbiguousCallXYZ, errWrongNumberOfArguments,
+    errWrongNumberOfArgumentsInCall,
```

Comment:
I don't add these messy ``err`` enum fields anymore, instead I use errGenerated or some overload of the error reporting procs that use that.